src/js/twitter.js

```diff
@@ -10,3 +12,47 @@
   js.src = p + '://platform.twitter.com/widgets.js'
   fjs.parentNode.insertBefore(js, fjs)
 })(document, 'script', 'twitter-wjs')
+
+const twitter_filter = (twitter_id) => {
+  if (!twitter_id) return null
+  const twitter_handles = []
+  const orgs = document.getElementsByClassName('org-info')
+  console.log('Running')
+  for (let i = 0; i < orgs.length; i++) {
+    const org_info = orgs[i].children
+    const children = [].slice.call(org_info)
+    const twitter_url = children.filter(
+      x => x.nodeName === 'A' && x.host === 'twitter.com'
+    )
+
+    if (twitter_url.length !== 0) {
+      const twitter_account = twitter_url[0].pathname.substr(1)
+      twitter_handles.push(twitter_account)
+    }
+  }
+
+  if (document.getElementById(twitter_id) !== null) {
+    const tweet_authors = document
+      .getElementById(twitter_id)
+      .contentWindow.document.getElementsByClassName(
+        'TweetAuthor-screenName Identity-screenName'
+      )
+
+    if (tweet_authors.length > 0) {
+      for (var k = 0; k < tweet_authors.length; k++) {
+        if (
+          twitter_handles.indexOf(tweet_authors[k].innerHTML.substr(1)) > -1
+        ) {
+          const tweet = getNodesDeep(tweet_authors[k], 5)
+          tweet.style.display = 'none'
+        }
+      }
+    } else {
+      setTimeout(twitter_filter(twitter_id), 1000)
```

Comment:
I think it needs to be `() => twitter_filter(twitter_id)`

Reply:
Yes. As is, `twitter_filter` will run immediately (without the 1 second delay) because `setTimeout` will be trying to call the result of the function.